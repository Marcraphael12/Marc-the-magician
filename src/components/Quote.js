import './Quote.scss';

const Quote = () => (
    <p className="Quote">
      "Mathematics is not about numbers, equations, computations or algorithm:
      It is about understanding" - Williams Paul Thurston.
    </p>
);

export default Quote;