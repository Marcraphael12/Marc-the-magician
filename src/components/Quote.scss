.Quote {
  font-size: 1.5rem;
  font-weight: 100;
  font-style: italic;
  width: 100%;
  height: calc(100% - 100px);
}
