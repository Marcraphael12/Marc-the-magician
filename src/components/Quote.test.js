/* eslint-disable no-unused-vars */
import renderer from 'react-test-renderer';
import {
  render,
} from '@testing-library/react';
import Quote from './Quote';

it('Quote rendering test', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

render(<Quote />);
