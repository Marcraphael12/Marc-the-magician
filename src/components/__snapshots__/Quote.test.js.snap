// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote rendering test 1`] = `
<p
  className="Quote"
>
  "Mathematics is not about numbers, equations, computations or algorithm: It is about understanding" - Williams Paul Thurston.
</p>
`;
